src/pages/proposals/create/StepDescription.vue

```diff
@@ -35,7 +35,14 @@ export default {
 
   computed: {
     nextDisabled () {
-      if (this.title.length > 0 && this.$sanitize(this.description, { allowedTags: [] }).length < DESCRIPTION_MAX_LENGTH && this.title.length <= TITLE_MAX_LENGTH) {
+      if (this.$store.state.proposals.draft.edit) {
+        if (this.$sanitize(this.description, { allowedTags: [] }).length < DESCRIPTION_MAX_LENGTH) {
```

Comment:
can you compute this value to something like `sanitizedDescription` so you can reuse it and it removes clutter?

Reply:
@evgenibir

Reply:
@arsenijesavic ready!